contents/handbook/small-teams/marketing/objectives.mdx

```diff
@@ -1,27 +1,29 @@
-* **Objective**:
-    * Become an authority and vital resource on building successful products.
+* **Goal**:
+    * Get twice as good at everything we’re already doing.
 
-* **Goals:**
-    * Nail Twitter for content distribution (**Owner:** Andy)
-    * Make PostHog for Startups the default choice for technical founders (**Owner:** Joe)
-    * Scale the newsletter to thousands of engaged subscribers (**Owner:** Andy)
-    * Create a hub of non-PostHog product engineering content (**Owner:** Ian)
-    * Ship printed PMF guide with a distribution plan (**Owner:** Lottie)
-    * Improve the docs and tutorials (**Owner:** Lior)
+*  **Projects**
+   * Win half of the W24 batch and double the size of the startups program (Joe)
+   * Double the number of newsletter subs again while maintaining 45% open rate (Ian)
+   * Every time you search ‘alternative to X’, PostHog appears high up in Google Search (Andy)
+   * Every time you search for content related to PostHog’s products, we appear high up in Google Search (Lior)
+   * Ship new high end merch that people post about online (Lottie)
+   * Do billboards so distinctive that people post them online (Lottie)
```

Comment:
just for context, this worked for brex because they coupled it with a cold outbound campaign to people based in SF near where they showed billboards

of course, it could just work regardless..!